source/_posts/2021-07-07-release-20217.markdown

```diff
@@ -56,8 +56,7 @@ Alright, that's it! Enjoy the release!
 
 ## New entity: Select
 
-In this release we welcome the `select` entity to the Home Assistant family. The
-select entity is family of the dropdown helper (also known as
+In this release, we welcome the `select` entity to the Home Assistant family. The select entity is a close relative of the dropdown helper (also known as
```

Comment:
"relative" was removed because of feedback, as "relative" was interpreted wrongly by some non-native speakers. Hence it was replaced by "family" during the beta.

Reply:
I'm a non-native speaker and couldn't find this usage in a dictionary. That's why I changed it. Relative itself might be ambiguous but "close relative" should be more easily understood than the uncommon usage of "family".